.github/workflows/check-binary-size.yml

```diff
@@ -6,9 +6,12 @@ name: Check binary size
 
 on:
   pull_request_target:
-    # HACK(jubilee): something broke the distributed LLVM libso and I don't know what.
-    branches: []
-#      - master
+    branches:
+      - master
```

Comment:
What is the motivation for the change to this filter? This sure looks like adding a value to an *empty* array, which seems to increase the number of possibilities. I'm not entirely sure if I understand what this did anymore and the GitHub Actions docs don't provide enough info for me to understand how Actions interprets an empty array here either, so I am not convinced this change is good.

Reply:
Ah, the answer is that the filter that matches nothing instead matches everything. Great.

Reply:
> @Marcono1234 You can tell GitHub that if the empty array doesn't mean "empty" then they have a bug.

I don't have any direct connection to GitHub. I can only tell them that the same way you can, through https://github.com/orgs/community/discussions I assume. There is actually an existing discussion about it: https://github.com/orgs/community/discussions/158135 (not sure if other duplicate ones exist as well)